drivers/gpio/Kconfig.ht16k33

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2019 Henrik Brix Andersen <henrik@brixandersen.dk>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig GPIO_HT16K33
+	bool "HT16K33 keyscan driver"
+	depends on (HT16K33_KEYSCAN && GPIO)
```

Comment:
Nit: Could skip the `()` (unless it's deliberate)

Reply:
@ulfalizer Fixed, thanks.